packages/gatsby-plugin-mdx/README.md

```diff
@@ -203,44 +203,39 @@ Here's a color picker!
 <SketchPicker />
 ```
 
-If you want to allow usage of a component from anywhere, add it to the `globalScope`
-field in the options inside `gatsby-config.js`:
+#### Shortcodes
+
+If you want to allow usage of a component from anywhere (often referred to as a
+shortcode), you can pass it to the
+[MDXProvider](https://www.gatsbyjs.org/docs/mdx/customizing-components/).
 
 ```js
-// gatsby-config.js
-module.exports = {
-  plugins: [
-    {
-      resolve: `gatsby-mdx`,
-      options: {
-        globalScope: `
-          import { SketchPicker } from "react-color";
+// src/components/layout.js
+import React from "react"
+import { MDXProvider } from "@mdx-js/react"
+import { YouTube, Twitter, TomatoBox } from "./ui"
```

Comment:
should we remove the all the unknown `TomatoBox` and exchange it to a well-known "silly" placeholder names like `Foo`, `Bar`, `FooBar`?

Reply:
Please no.....

Reply:
it takes me many minutes to find out that there is no code anywhere available for a `/ui/TomatoBox` and also no Service `TomatoBox` like `YouTube` or `Twitter`. 

if you don't like the naming `FooBar` then the example `TomatoBox` should be have a naming which shows directly without googling that it is a placeholder or example name.

Reply:
Hi @muescha -- let's keep this scoped to the PR (updating the references to `gatsby-mdx`). If you feel strongly about this, please open it for discussion in a new PR. Thanks!